Formula/uncrustify.rb

```diff
@@ -25,7 +25,10 @@ class Uncrustify < Formula
   # patches from https://github.com/uncrustify/uncrustify/pull/3178
   # and https://github.com/uncrustify/uncrustify/pull/3179 that fix build failures
   # can be removed for 0.74.0
-  patch :DATA
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/master/uncrustify/uncrustify-0.73.0.patch"
```

Comment:
```suggestion
    url "https://raw.githubusercontent.com/Homebrew/formula-patches/52cd74a3a21841bd6dd6bf2d1e95af8e5e5dfa16/uncrustify/uncrustify-0.73.0.patch"
```
Let's use a stable URL here.